assets/base.css

```diff
@@ -561,6 +566,10 @@ h6:empty {
   font-family: inherit;
 }
 
```

Comment:
From what I understand, inputs shouldnt be changing sizes. Is that correct? https://screenshot.click/28-24-4m7d4-kfyt8.mp4

Reply:
Inputs will change in scale if `body_text` setting is changed. In fact, all padding, margin, height, width in the theme scales with `body_text` setting